src/components/aside/aside.js

```diff
@@ -0,0 +1,40 @@
+import "./aside.css"
+import logo from "../../assets/images/logo.png"
+
+function Aside () {
+    return(
+        <aside className="aside">
+            <div className="aside__header">
+                <img src={logo}/>
+                <p>BeLoud</p>
```

Comment:
You can add a cursor pointer & a link here if required

Reply:
Yeah will do it.